test/surface_test.py

```diff
@@ -990,16 +990,10 @@ def todo_test_get_colorkey(self):
         self.assertIsInstance(ck, pygame.Color)
         self.assertEqual(ck, colorkey)
 
-    def todo_test_get_height(self):
-
-        # __doc__ (as of 2008-08-02) for pygame.surface.Surface.get_height:
-
-        # Surface.get_height(): return height
-        # get the height of the Surface
-        #
-        # Return the height of the Surface in pixels.
-
-        self.fail()
+    def test_get_height(self):
+        surf = pygame.Surface((2,4))
+        height = surf.get_height()
+        self.assertEqual(height, 4)
```

Comment:
Hi! 
Thanks for having a go at this, my main suggestion to make this test a bit more comprehensive is to test more values in a loop, just in case the function does anything weird in edge case situations e.g. :

```suggestion
    def test_get_height(self):
        sizes = ((1,1), (119, 10), (1000,1000))  # Could add a few more ideas here 
        for width, height in sizes:
            surf = pygame.Surface((width, height))
            found_height = surf.get_height()
            self.assertEqual(height, found_height)
```